### DISCUSSION TITLE
When I use rollup to build the gltf-transform/core package, I get this error and I don't know how to fix it @gltf-transform/core

### DISCUSSION THREAD

**Discussion by Sruimeng** [donmccurdy/glTF-Transform#531] [Q&A] [answered]
```
like this
[!] Error: Unexpected token
node_modules/_@gltf-transform_core@2.0.5@@gltf-transform/core/dist/core.modern.js (1:61744)
........
2: //# sourceMappingURL=core.modern.js.map
Error: Unexpected token
```
 I get this error and I don't know how to fix it

**Comment by donmccurdy** [comment, marked as answer]
Hi! You'll need to share more information for me to answer this question.

- Are you using Rollup just to build the `@gltf-transform/core` package specifically? Or to build some other project that depends on it? 
- What specific commands are you running, and what is your rollup config?

If you're trying to build _this_ project by itself, the commands are documented in [Contributing](https://gltf-transform.donmccurdy.com/contributing.html):

```
yarn install
yarn run dist
```
